library_test.go

```diff
@@ -55,3 +55,54 @@ func TestRemoveTracksFromLibrary(t *testing.T) {
 		t.Error(err)
 	}
 }
+
+func TestUserHasAlbums(t *testing.T) {
+	client, server := testClientString(http.StatusOK, `[ false, true ]`)
+	defer server.Close()
+
+	contains, err := client.UserHasAlbums("47DiM7xqFvwplamrB1GQJo", "73TmwDD6mBOZh6sF9sKXZo")
+	if err != nil {
+		t.Error(err)
+	}
+	if l := len(contains); l != 2 {
+		t.Error("Expected 2 results, got", l)
+	}
+	if contains[0] || !contains[1] {
+		t.Error("Expected [false, true], got", contains)
+	}
+}
+
+func TestAddAlbumsToLibrary(t *testing.T) {
+	client, server := testClientString(http.StatusOK, "")
+	defer server.Close()
+
+	err := client.AddAlbumsToLibrary("47DiM7xqFvwplamrB1GQJo", "73TmwDD6mBOZh6sF9sKXZo")
+	if err != nil {
```

Comment:
This doesn't test much other than that we return a nil error.

I wonder if we can make some assertions to verify that the correct endpoint was hit with the correct data.

Reply:
True, i'll have a crack at this one when i'm back from ski holiday ;) ⛷🎉